### PR TITLE
Move tests to tests/unit or tests/integration 

### PR DESCRIPTION
**Pull request by jperez999**
This moves tests to either tests/unit or tests/integration - to be consistent with other repo's in merlin like merlin-core, nvtabular and merlin-systems.

### CODE REVIEW

**Comment by jperez999** [comment]
rerun tests

**Comment by jperez999** [comment]
rerun tests

**Review by benfred — APPROVED** [review]

File: ci/test_integration.sh [hunk 1]
```diff
@@ -17,15 +17,4 @@
...
-# Get latest models version
-cd /models/
-git pull origin main
-
-container=$1
-
-## Tensorflow training container
-if [ "$container" == "merlin-tensorflow-training" ]; then
-    ## TODO: Remove this when it would become part of the container
-    pip3 install testbook
```
Comment (benfred) [inline ci/test_integration.sh:29]: do we have testbook and jupyterlab  on the container now ?

File: ci/test_integration.sh [hunk 2]
```diff
@@ -17,15 +17,4 @@
 #!/bin/bash
 set -e
 
-# Get latest models version
-cd /models/
-git pull origin main
```
Comment (benfred) [inline ci/test_integration.sh:22]: good catch on removing the 'git pull' - I thought we had got rid of all these

File: ci/test_integration.sh (hunk 1 above)
Comment (jperez999) [reply ci/test_integration.sh:29]: no they don't currently.

**Pull request merged**